CHANGELOG.rst

```diff
@@ -65,6 +65,19 @@ Added
 
   Contributed by @Kami.
 
+* Add support for compressing the payloads which are sent over the message bus. Compression is
+  disabled by default and user can enable it by setting ``transport.compression`` config option
```

Comment:
Looks like it is `[messaging]` section and `compression` setting

Reply:
Good catch, will fix it.

Reply:
91a61a20860566d4886c13f1cff48a0e18ba2342